cumulusci/cli/cci.py

```diff
@@ -1131,26 +1131,19 @@ def flow_run(config, flow_name, org, delete_org, debug, o, skip, no_prompt):
     if delete_org and not org_config.scratch:
         raise click.UsageError("--delete-org can only be used with a scratch org")
 
-    flow_config = config.project_config.get_flow(flow_name)
+    coord = config.get_flow(flow_name)
```

Comment:
No benefit from abbreviating, call it coordinator so it's obviously not coordinate